modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -179,10 +179,6 @@ func (suite *KeeperTestSuite) TestDistributeFee() {
 			suite.chainA.GetSimApp().IBCFeeKeeper.DistributePacketFees(suite.chainA.GetContext(), forwardRelayer, reverseRelayer, []types.PacketFee{packetFee, packetFee})
 
 			if tc.expPass {
-				// there should no longer be a fee in escrow for this packet
-				found := suite.chainA.GetSimApp().IBCFeeKeeper.HasFeeInEscrow(suite.chainA.GetContext(), packetID)
```

Comment:
This is removed as `DeleteFeesInEscrow()` is called at the ibc_module.go level [here](https://github.com/cosmos/ibc-go/blob/ics29-fee-middleware/modules/apps/29-fee/ibc_module.go#L243) and [here](https://github.com/cosmos/ibc-go/blob/ics29-fee-middleware/modules/apps/29-fee/ibc_module.go#L270) for timeout.